### PR TITLE
USWDS-Site: Add changelog for alert alignment fix [#5636]

### PR DESCRIPTION
**Pull request by amyleadem**
# Summary

Add a changelog for PR #5636

>[!important]
> We need to update the changelog dates before merge.

>[!important]
> ~We need to update uswds branch before merge.~ Done

## Related issue

Related to uswds/uswds#5636

## Preview link

- [Alert component page](https://federalist-ead78f8d-8948-417c-a957-c21ec5617a57.sites.pages.cloud.gov/preview/uswds/uswds-site/al-changelog-5636/components/alert/#latest-updates)
- [Site alert component page](https://federalist-ead78f8d-8948-417c-a957-c21ec5617a57.sites.pages.cloud.gov/preview/uswds/uswds-site/al-changelog-5636/components/site-alert/#latest-updates)

### CODE REVIEW

File: _data/changelogs/component-alert.yml [hunk 1]
```diff
@@ -2,6 +2,17 @@ title: Alert
 type: component
 changelogURL:
 items:
+  - date: NNNN-NN-NN
+    summary: |
+      Fixed a bug that caused `$theme-site-margins-width` to unexpectedly adjust the alignment inside the alert and site alert components.
```
Comment (mejiaj) [inline _data/changelogs/component-alert.yml:7]: **polish:** Minor suggestion for plain language.

```suggestion
      Fixed a bug that caused `$theme-site-margins-width` to unexpectedly change the alignment inside alert and site alert components.
```

File: _data/changelogs/component-alert.yml [hunk 2]
```diff
@@ -2,6 +2,17 @@ title: Alert
...
+    summaryAdditional: |
+      Alignment on the alert and site alert components will likely shift from this change.
```
Comment (mejiaj) [inline _data/changelogs/component-alert.yml:9]:
```suggestion
      Alignment of alert and site alert components will likely shift from this change.
```

File: _data/changelogs/component-alert.yml [hunk 3]
```diff
@@ -2,6 +2,17 @@ title: Alert
...
+      Confirm that your implementation of the component aligns as expected.
+    isBreaking: true
+    affectsStyles: true
+    githubPr: 5636
+    githubRepo: uswds
+    versionUswds: N.N.N
```
Comment (mejiaj) [inline _data/changelogs/component-alert.yml:15]:
```suggestion
    versionUswds: 3.9.0
```

File: _data/changelogs/component-site-alert.yml [hunk 4]
```diff
@@ -2,6 +2,17 @@ title: Site alert
...
+    summary: |
+      Fixed a bug that caused `$theme-site-margins-width` to unexpectedly adjust the alignment inside the alert and site alert components.
+    summaryAdditional: |
+      Alignment on the alert and site alert components will likely shift from this change.
+      Confirm that your implementation of the component aligns as expected.
+    isBreaking: true
+    affectsStyles: true
+    githubPr: 5636
+    githubRepo: uswds
+    versionUswds: N.N.N
```
Comment (mejiaj) [inline _data/changelogs/component-site-alert.yml:15]:
```suggestion
    versionUswds: 3.9.0
```

File: package.json [hunk 5]
```diff
@@ -98,6 +98,6 @@
   "snyk": true,
   "dependencies": {
     "@uswds/compile": "1.2.0",
-    "@uswds/uswds": "3.8.2"
+    "@uswds/uswds": "github:uswds/uswds#al-alert-icon-alignment"
```
Comment (mejiaj) [inline package.json:101]: **issue**: We should revert this change.

**Review by mejiaj — COMMENTED** [review]
Thanks @amyleadem. Added small suggestions for plain language.

File: package.json (hunk 5 above)
Comment (amyleadem) [reply package.json:101]: Leaving it in for now while uswds/uswds#5636 is still in review. This way we can provide a demo of the on site.

**Comment by amyleadem** [comment]
Suggestions look good @mejiaj. Ready for your re-review

**Review by mejiaj — APPROVED** [review]
